hwconf/hw_Little_FOCer.h

```diff
@@ -22,7 +22,7 @@
 
 #include "drv8323s.h"
 
-#define HW_NAME                 "Little FOCer"
+#define HW_NAME                 "Little_FOCer"
```

Comment:
It'd also be great to add _v10 here or something so as not to mix it up with the popular version 0.9

Reply:
There is no v0.9 for the little focer.